influxdb/package/install.sls

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/map.jinja" import influxdb with context %}
+
+influxdb/package/install:
+  pkg.installed:
+    - name: {{ influxdb.pkg.name }}
```

Comment:
Maybe a fix for the opensuse failure, test with refresh in pkg.installed
```
influxdb/package/install:
  pkg.installed:
    - name: {{ influxdb.pkg.name }}
    - refresh: true
```

Reply:
It seems that `influxdb` has been removed : 

```sh
6f14d870814f:/etc/zypp/repos.d # zypper search influx
Loading repository data...
Reading installed packages...

S | Name                                  | Summary                                                           | Type      
--+---------------------------------------+-------------------------------------------------------------------+-----------
  | golang-github-influxdata-usage-client | A Golang client for speaking to the Anonymous Usage Reporting API | package   
  | golang-github-influxdata-usage-client | A Golang client for speaking to the Anonymous Usage Reporting API | srcpackage
  | pcp-export-pcp2influxdb               | Performance Co-Pilot tools for exporting PCP metrics to InfluxDB  | package   
  | python2-influxdb                      | InfluxDB client                                                   | package   
  | python3-influxdb                      | InfluxDB client                                                   | package   ```

Reply:
@n-rodriguez Could we use one of the experimental or community packages instead?

* https://software.opensuse.org/package/influxdb

Reply:
it works